<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pong_Class</class>
 <widget class="QWidget" name="Pong_Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Titre">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Pong</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Play">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Jouer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Scores">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scores</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
